### ISSUE TITLE
App crashes after login

### ISSUE THREAD

**Issue by npcode** [zulip/zulip-android#19]
I modified and rebuilt the zulip app to connect my own zulip server. Then I installed it and ran it in my phone.

When I put my email address and password and click login button, the app crashes after showing a spinner for a while.

My zulip server seems to log the access from the app successfully.
```
2015-11-27 05:25:58,093 INFO     10.***.***.***    POST    200 121ms (mem: 5ms/7) (db: 24ms/12q) /api/v1/register (eungjun.yi@navercorp.com via Android)
```

ps. My zulip server is configured to use LDAP.

**Comment by raicabogdan** [comment]
If you still require help on this, could you include a logcat of the error ?

**Comment by lfaraone** [comment]
Fixed in 6c91ca7d
